packages/cloud_security_posture/changelog.yml

```
@@ -1,9 +1,9 @@
 # newer versions go on top
-- version: "1.1.0"
+- version: "1.0.9"
```

I think it should be like:
```suggestion
- version: "1.0.9"
  changes:
    - description: Support spaces
      type: enhancement
      link: https://github.com/elastic/integrations/pull/4804
- version: "1.1.0"
    - description: Introduce CSPM
      type: enhancement
      link: https://github.com/elastic/integrations/pull/4752
```